src/simulations/schemas.py

```diff
@@ -54,6 +54,12 @@ class Metabolomics(Schema):
     uncertainty = fields.Float(required=True)
 
 
+class Proteomics(Schema):
+    identifier = fields.String(required=True)  # UniProt code
+    measurement = fields.Float(required=True)  # mmol/gDw
+    uncertainty = fields.Float(required=True)  # mmol/gDw
+
```

Comment:
Would it make sense to add a namespace here too, like in Metabolomics and Fluxomics?

Reply:
@carrascomj as we are not using that field in any of the `adapter` or `driven` functions I would avoid it for keeping the code briefer. If we ever need it then of course we can introduce it.